docs/modules/ROOT/pages/how-tos/testing/build/snyk.adoc

```diff
@@ -1,6 +1,54 @@
-= Enabling Snyk
+= Enabling a Snyk task
 
-While a Snky test is available to run a build time in the default pipelines, additional configuration is needed to enable the test. This is an example of a build-time test that requires the configuration of a custom secret to be able to run.
+The Snyk test is available to run at build time in the default pipelines, but it requires additional configuration to enable. This procedural example illustrates a build-time test that requires the configuration of a custom secret.
 
-+
-NOTE:  We may want to consider generalizing this topic.
\ No newline at end of file
+The `sast-snyk-check` task uses the Snyk Code tool to perform static application security testing (SAST). 
+Specifically, the Snyk check scans an application's source code for potential security vulnerabilities, 
+including SQL injection, cross-site scripting (XSS), and code injection attack vulnerabilities.
+
+NOTE: You can run a Snyk task only if you have a Snyk token stored in a namespace secret. Ensure that the name of your secret is included in the *snyk-secret* pipeline parameter.
+
+.Procedure
+
+. Register for a Snyk account or log in at https://app.snyk.io/.
+. Get a Snyk token.
+
+.. In the lower left of the home page, click your name, then select *Account settings*.
+
+.. From the Account Settings page, select *General*, which is the default, then *Auth Token*.
+
+.. Under the *Auth Token* section, click *Click to View* to see the *KEY* value of the automatically generated token.
```

Comment:
Is there Snyk documentation that we can/should point to? I assume that this is relatively stable, but in case it isn't do we want to keep it updated?

Having the documentation all in one place is beneficial ... so I am just asking the question, not requesting a change.

Reply:
We have the link to Snyk docs in Additional resources. Let me know if that works.